docusaurus/docs/engineering/start-developing/boba-frontend.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 5
+---
+
+# Install BobaFrontend
+
+:::warning
+This codebase requires Node **16**. If you're using [Node Version Manager](https://github.com/nvm-sh/nvm), you can switch to version 16 by running `nvm use 16` in your terminal.
+
+You can check the current Node version by running `node -v`.
+:::
+
+## Clone project
+
+In a terminal, run the following commands:
+
+```bash showLineNumbers
+# Clone the codebase from github
+git clone https://github.com/BobaBoard/boba-frontend.git
+# Enter the codebase directory
+cd boba-frontend
+# Install all necessary dependencies
+yarn install
+```
+
+`boba-frontend` does not currently require any special env set up.
+
+## Develop
+
+:::TODO
+Move these instructions to the `boba-frontend` section, similar to how it's done in the `boba-backend` install instructions.
+:::
+
+### Connect to a local `boba-backend` (default Realm)
+
+To connect to a local `boba-backend` instance, make sure that the instance is up, then run `yarn run dev`. Going to
+`http://localhost:3000` will show the default Realm (`twisted-minds`).
```

Comment:
does this actually work now? we should make sure it does if we're adding this instruction